services/auth/src/signaturevalidator.js

```diff
@@ -137,7 +137,7 @@ const limitClientWithExt = function(credentialName, issuingClientId, accessToken
       res.expires = cert_expires;
     }
 
-    res.scopes = scopes = expandScopes(cert.scopes);
+    res.scopes = scopes = cert.scopes;
```

Comment:
iiuc we don't expand here anymore because we always do it to the result of this function anyway. We can remove `expandScopes` from the arguments of this function and where it is called in that case I believe